h/views/activity.py

```diff
@@ -319,7 +319,8 @@ def toggle_tag_facet(request):
         tag_facets = _tag_facets(request, parsed_query)
         tag_facets.remove(tag)
         if tag_facets:
-            parsed_query['tag'] = tag_facets
+            for tag_facet in tag_facets:
+                parsed_query['tag'] = tag_facet
```

Comment:
Shouldn't this add a tag facet to the parsed query rather than replacing it?

If I do the following, I get an unexpected result:

1. Create an annotation with 3 tags
2. Go to /users/username/search and click each of the three tags in the sidebar in succession. This should result in `username:<name> tag:<first> tag:<second> tag:<third>` in the search input
3. Remove the last tag from the query by clicking it again in the sidebar

**Expected** - Only the last tag is removed from the query
**Actual** - The first tag added is also removed from the query.